Rasa_Bot/actions/requirements-actions.txt

```diff
@@ -1,3 +1,3 @@
 python-dateutil==2.8.1
-git+https://github.com/PerfectFit-project/virtual-coach-db
+git+https://github.com/PerfectFit-project/virtual-coach-db@add_new_user
```

Comment:
@svenvanderburg If the database runs on localhost, then it cannot be accessed from inside the Docker containers created via the root docker-compose on Windows. Instead, it can be accessed via host.docker.internal. For now, I just created a new db branch where I changed the helper function get_db_session() in that it uses host.docker.internal instead of localhost. This helper function is used by the rasa custom actions. But I am not sure if this host.docker.internal also works on linux. What would be a good way to solve this?

Reply:
This should work on both windows and mac, see [this](https://stackoverflow.com/questions/24319662/from-inside-of-a-docker-container-how-do-i-connect-to-the-localhost-of-the-mach), not on linux.

I suggest that you use a .env file to configure the database url on the client-side (so on the rasa side). So add an environment variable called DB_HOST, and load that into the rasa actions code. 

That also means updating https://github.com/PerfectFit-project/virtual-coach-db/blob/25d51eedbebe67fc3a49b66be7442243b744663a/helper/helper.py#L11
so that it accepts parameters (i.e. db_host, db_password, db_user, but falls back to the constants defined there).